x/auth/simulation/decoder.go

```diff
@@ -15,15 +15,17 @@ import (
 func DecodeStore(cdc *codec.Codec, kvA, kvB tmkv.Pair) string {
```

Comment:
Note to self/others: 

This is something I need to look into. We already have a few modules migrated over to protobuf and simulations are passing. Yet, all the module's `DecodeStore` functions still utilize Amino...